Quadriga/src/main/webapp/WEB-INF/tiles-defs.xml

```diff
@@ -74,13 +74,22 @@
 		<put-attribute name="currentPage" value="home" />
 	</definition>
 
+   <!-- checks -->
+   <definition name="check" extends="base.definition.nonav">
+       <put-attribute name="content" value="/WEB-INF/views/checks.jsp" />
+	</definition>
 
-    <!-- checks -->
-    <definition name="check" extends="base.definition.nonav">
-        <put-attribute name="content" value="/WEB-INF/views/checks.jsp" />
+    <!-- result -->
+    <definition name ="auth/statusTransformation" extends="base.definition.nonav">
+        <put-attribute name="content" value="/WEB-INF/views/transformation/statusTransformation.jsp" />
     </definition>
-    
-
+    <definition name="auth/successTransformation" extends="base.definition.nonav">
+        <put-attribute name="content" value="/WEB-INF/views/transformation/successTransformation.jsp" />
+    </definition>
+	<definition name="auth/failureTransformation" extends="base.definition.nonav">
+        <put-attribute name="content" value="/WEB-INF/views/transformation/failureTransformation.jsp" />
+    </definition>
```

Comment:
those definitions should have names according to the convention we use (e.g. auth/transformation/viewname).

also, please put them with the other transformation views.